accounts/account_manager.go

```diff
@@ -101,6 +101,17 @@ func (am *Manager) firstAddr() ([]byte, error) {
 	return addrs[0], nil
 }
 
+func (am *Manager) getKey(addr []byte, keyAuth string) (*crypto.Key, error) {
+	if len(addr) == 0 {
```

Comment:
This is not a good idea. `Unlock(nil, ...)` should never work.
This change is also unrelated to the account import/export.